RequestManagementSystem/Client/Request.py

```diff
@@ -167,8 +167,8 @@ def _notify( self ):
       # # All operations Done -> Done
       elif opStatus == "Done" and self.__waiting == None:
         rStatus = "Done"
-        self._Error = ''
-    self.Status = rStatus
+        self.Error = ''
+    self._Status = rStatus
```

Comment:
While the two other changes are correct, this one is not. You want to go through the setter to update the time and things like this